### DISCUSSION TITLE
Whatsapp autoreply message

### DISCUSSION THREAD

**Discussion by jem555** [LiveHelperChat/livehelperchat#1982] [Q&A]
Hi!. I'm trying set up an automatic reply as welcome message to whatsapp visitor after first message. What should I do?
Thanks in advance!

**Comment by wishesbeast** [comment]
You should try [an whatsapp](https://latestgbapps.com/an-whatsapp-apk/) for advanced features and themes. This app will provide auto reply text in modified form with best features.

**Comment by remdex** [comment]
You can just setup a bot with a default message after that just transfer chat again to pending state :)

**Reply by jem555** [reply]
Hi!. the thing is there has to be a bot to send whatsapp messages out. A new bot will never make it if I choose "send text" instead of "Rest Api". What'd be the right way to do it?
Thanks!

**Reply by remdex** [reply]
If you are using official integration fbmessenger all has to work out of the box. Today I added support for autoresponder where you can setuo simple welcome message. Bot scenario which I told previously also has to work. You don't need to setup any additional rest api calls...

**Reply by remdex** [reply]
I have added a few sections at the beginning of this document...
https://doc.livehelperchat.com/docs/auto-responder

**Reply by jem555** [reply]
Excellent! Thanks for your help. I updated, then I set up an autoresponder and it worked out of the box, but only the main text box, not the others that works with timeouts.
I'd like to know if there's a chance to set up a message on chat acceptance or assign just to inform whatsapp visitors who operator is (as you know whatsapp doesn't show the operator).

**Reply by remdex** [reply]
I ques you missed to setup workflow cronjob every 5 seconds... https://doc.livehelperchat.com/docs/auto-responder

Also there is new event you have to setup. Everything in the docs

**Comment by Maverick990** [comment]
To set up an automatic welcome message on WhatsApp, follow these steps:

WhatsApp Business API: Get approved to use the WhatsApp Business API.

Provider: Choose a WhatsApp Business Solution Provider for setup.

Message Templates: Create a friendly welcome template for automated replies.

Integration: Connect API with your systems or CRM.

Testing: Ensure the welcome message works correctly.

Remember to use "WhatsApp" following their branding guidelines.
Or you can use [blue whatsapp](https://apkwa.net/blue-whatsapp-plus/) or [whatsapp plus](https://apkwa.net/whatsapp-plus/).

**Reply by mhdhhemant** [reply]
Hi Can you help me in detail, how can i set an incoming Webhook to get the message from WhatsApp user, and reply back through the Bot. i want to hit the third party API to send message back to visitor , and to receive i am giving the API to third party, which will hit my webhook.Please help.

**Comment by mhdhhemant** [comment]
Hi Can you help me in detail, how can i set an incoming Webhook to get the message from WhatsApp user, and reply back through the Bot. i want to hit the third party API to send message back to visitor , and to receive i am giving the API to third party, which will hit my webhook.Please help.
